.github/workflows/tests.yml

```diff
@@ -62,43 +70,22 @@ jobs:
           from os.path import basename
           from re import compile
           main = '^main$'
-          release = '^v\d+\.\d+\.\d+$'
+          release_starts_r = 'release_\d+\_\d+\_\d+$'
```

Comment:
We want release branches to be named `release_0_0_9` so that they don't conflict with tags (ReadTheDocs gets confused).